api/models/question-attempt.js

```diff
@@ -0,0 +1,31 @@
+const {
+  Model
+} = require('sequelize');
+const JsonApiModel = require('base/jsonApiModel');
+const SerializerOpts = require('serializer-opts/question-attempt');
+
+class QuestionAttempt extends JsonApiModel {
+  static get serializerOpts() {
+    return SerializerOpts(this);
+  }
+
+  static associate({ examAttempts, questions, questionAttempts }) {
+    questionAttempts.belongsTo(examAttempts);
+    examAttempts.hasMany(questionAttempts);
+    questionAttempts.belongsTo(questions);
+    questions.hasMany(questionAttempts);
```

Comment:
won't be needing this, and because we automatically include first level relations this can cause a problem as well